xml/hwbp_registers.xml

```diff
@@ -189,7 +189,12 @@
         <field name="m" bits="6" access="R/W" reset="0">
             When set, enable this trigger in M mode.
         </field>
-        <field name="0" bits="5" access="R" reset="0" />
+        <field name="hit" bits="5" access="R/W" reset="0">
+            When this trigger matches, this optional bit becomes set. The
+            trigger's user can use it to see which trigger(s) matched, and is
```

Comment:
this is better, but what happens if the trigger's non-owner reads it, or tries to write it to 0/1?

Reply:
The same as all the other bits. The definition for `dmode` says:
            0: Both Debug and M Mode can write the {\tt tdata} registers at the
            selected \Rtselect.

            1: Only Debug Mode can write the {\tt tdata} registers at the
            selected \Rtselect.  Writes from other modes are ignored.

            This bit is only writable from Debug Mode.

I believe that answers your questions.

Reply:
This is orthogonal to the PR, but are the non-owners even allowed to read the \ttdata (or would it read 0 if it wasn't the owner)? It's not currently specified, not sure what the intention was.